src/sip.c

```diff
@@ -598,15 +619,13 @@ sip_parse_msg_media(sip_msg_t *msg, const u_char *payload)
                      * and port of the stream.
                      */
                     // Create a new stream with this destination address:port
-                    if (!call_msg_is_retrans(msg)) {
-                        if (!rtp_find_call_stream(call, src, dst)) {
-                            // Create RTP stream
-                            call_add_stream(call, stream_create(media, dst, PACKET_RTP));
-                            // Create early RTCP stream
-                            dst.port++;
-                            call_add_stream(call, stream_create(media, dst, PACKET_RTCP));
-                        }
-                    }
+
+                    // Create RTP stream
+                    rtp_stream = stream_create(media, dst, PACKET_RTP);
+
+                    // Create RTCP stream
+                    rtcp_stream = stream_create(media, dst, PACKET_RTCP);
+                    rtcp_stream->dst.port++;
                 }
             }
         }
```

Comment:
The `call_msg_is_retrans` and ` rtp_find_call_stream` checks were designed to avoid creating more than once the same stream structure. For example on retransmission of the initial invite or after a authentication request.